### DISCUSSION TITLE
Potential Source for 1.13 Swimming Animation Code?

### DISCUSSION THREAD

**Discussion by Rozmir-Rohi** [Roadhog360/Et-Futurum-Requiem#400] [Ideas]
This may have been suggested before but recently I was looking back on some old mods and I found that the underwater swimming animation added by the Smart Moving mod is very similar to the 1.13 swimming animation.

Maybe some of the code for the swimming animation from this mod could be used to add the 1.13 swimming animation?

Smart Moving Mod Original Forum Post: https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1274224-smart-moving

Showcase of the mod (skip to 1:12 - 1:15 to see the underwater swimming animation): https://youtu.be/hjmmQ0aTAZQ

The mod requires the following to work:
-  Smart Moving and Smart Render (use jar files from inside zip): https://www.mediafire.com/?7fd7f4z2o266ri9

- Player API (use jar file from inside zip): https://www.dropbox.com/sh/wfd5j8p3hp6rhmd/AABcYFw4Cqqt-_W5GTz1Do0Ba/MC%201.7.10%20-%20Player%20API%20core%201.4.zip?dl=1

The Smart Moving mod and Smart Render mod are licensed under the GNU General Public License v3.0.

The Player API mod is licensed under the GNU Lesser General Public License v3.0.

**Comment by Roadhog360** [comment]
None of this is new information.

**Comment by Roadhog360** [comment]
This also appears to be suggesting that I add swimming, which is a duplicate of #155

**Comment by Roadhog360** [comment]
I also realized that it should be noted that you are linking to old versions of smart moving, and in a zip file in Dropbox for whatever reason

Here are better and more polished releases:
https://github.com/makamys/PlayerAPI
https://github.com/makamys/SmartMoving
